docs/initialization-steps-and-the-related-tracer.org

```diff
@@ -0,0 +1,624 @@
+#+AUTHOR: Pawel Szulc
```

Comment:
I think this file should go under `ouroboros-consensus/docs`, then it will also get the right code owners.

Reply:
👍 I'll move it there

Reply:
@coot this has been moved, reviewers removed, can you 👍 this ? :)